js/loader.js

```diff
@@ -35,7 +35,7 @@ var Loader = (function() {
 				// This is done after all the moduels so we can
 				// overwrite all the defined styls.
 
-				loadFile("css/custom.css", function() {
+				loadFile(config.customCss, function() {
```

Comment:
Hi there,

sorry for the late response. I had a look again at your suggestions and I'm still not sure if I get it right.

- fs.exists() can not be used at that point in the loader.js because it can not require NodeJs modules, it's javascript only. So the result would be a bigger refactoring of the loader or the place the custom.css is loaded.

- The internal "loadFile" method in the loader.js is already appling some sort of error handling:
```
stylesheet.onerror = function() {
	console.error("Error on loading stylesheet:", fileName);
	if (typeof callback === "function") {callback();}
};
```
So there will already be logged in the console that the file could not be found.

What would your solution look like?